src/WallEvaluatorColloid.h

```diff
@@ -83,6 +135,7 @@ class WallEvaluatorColloid
      */
     DEVICE void setDiameter(Scalar di, Scalar dj)
```

Comment:
There are no more diameters in HOOMD (see other codes). This needs to be removed, and the radius of the particle needs to be set as a parameter. So, you may want to have an `a_w` (for the wall) and an a for the particle as parameters. See
https://github.com/mphowardlab/azplugins/blob/azplugins-v1/src/PairEvaluatorColloid.h
for how it is done for two particles.